tests/Argu.Tests/Argu.Tests.fsproj

```diff
@@ -4,10 +4,16 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Tests.fs" />
-    <None Include="paket.references" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\Argu\Argu.fsproj" />
   </ItemGroup>
-  <Import Project="..\..\.paket\Paket.Restore.targets" />
+  <ItemGroup>
+    <PackageReference Include="FSharp.Core" VersionOverride="6.0.7" />
```

Comment:
can the version override go on Argu itself ? tests and samples should take latest?

Reply:
No, one of the test packages needed a higher `FSharp.Core` version. So, I'm taking the highest that more or less matches the SDK version.

Reply:
Yeah, the `implicit version` clashes with CPM. You will get a warning after `restore`, that you are specifying a version for `FSharp.Core` (implicitly) while using CPM. That is why I went with that `VersionOverride`, a tradeoff for sure.